GPU/Vulkan/TextureScalerVulkan.cpp

```diff
@@ -21,38 +21,45 @@
 #endif
 
 #include <algorithm>
-#include "gfx/gl_common.h"
 
-#include "GPU/Common/TextureScalerCommon.h"
-#include "GPU/Vulkan/TextureScalerVulkan.h"
+#include "Common/Vulkan/VulkanContext.h"
 #include "Common/ColorConv.h"
 #include "Common/Log.h"
 #include "Common/ThreadPools.h"
+#include "GPU/Common/TextureScalerCommon.h"
+#include "GPU/Vulkan/TextureScalerVulkan.h"
+
+// TODO: Share in TextureCacheVulkan.h?
```

Comment:
Uh wait, I'm tired, I read it as VulkanImage.cpp for some reason. Never mind my comment.